src/components/icons/Icon.jsx

```diff
@@ -14,11 +14,13 @@ const TYPE = {
   BOLD: 'bold',
   CAMERA: 'camera',
   CHECK: 'check',
+  CHECKED: 'checked',
```

Comment:
we already have 'check'. Lets use 'verified' here

Reply:
agree